pom.xml

```diff
@@ -82,8 +82,8 @@
     <version.com.google.gwt>2.8.2</version.com.google.gwt>
     <version.org.google.gwt-charts>0.9.9</version.org.google.gwt-charts>
     <version.com.google.gwt.gwtmockito>1.1.8</version.com.google.gwt.gwtmockito>
-    <version.com.google.jsinterop.base>1.0.0-beta-1</version.com.google.jsinterop.base>
-    <version.com.google.jsinterop>1.0.1</version.com.google.jsinterop>
+    <version.com.google.jsinterop.base>1.0.0-RC1</version.com.google.jsinterop.base>
```

Comment:
@jomarko Just curious: Clear to me why `<version.com.google.elemental2>1.0.0-RC1</version.com.google.elemental2>` should have this version. Why is the same for `jsinterop.base`? are they sharing the same lifecycle?

Reply:
@jomarko I analyzed the version, it seems that` com.google.elemental2` 1.0.0-RC1` uses `com.google.jsinterop.base 1.0.0-RC1 1.0.0-beta-3`, and `com.google.jsinterop.annotations 1.0.2` as you can see here https://github.com/google/elemental2/commit/527ebb5ce214d62806313555c48a7cda5f972827  (This commit was done AFTER `com.google.elemental2 1.0.0-RC1` release.
Then, best updated (and working) combination should be 
` <version.com.google.jsinterop.base>1.0.0-beta-3</version.com.google.jsinterop.base>
<version.com.google.jsinterop.annotations>1.0.1</version.com.google.jsinterop.annotations> 
<version.com.google.elemental2>1.0.0-RC1</version.com.google.elemental2>`

Anyway, considering we are near the release for dev-preview and we have a stable situation, I suggest to postpone or to undo this operation, because it could be have an high impact over all repos (and a `beta` version will remain, at all')